frontend/src/components/StudentProfile/GraduationStatus/GRADStatusForm.vue

```diff
@@ -819,7 +819,12 @@ export default {
           (value) => {
             return (
               this.editedGradStatus.program !== "1950" ||
-              !(this.editedGradStatus.schoolOfRecord.search(/^103.*/) >= 0)
+              !(
```

Comment:
Can we make this a shared function in common.js that takes a mincode? Similar to isPFSchool?

We should also be using the getter in app.js getSchoolById(schoolGUID).mincode to get mincodes so we aren't duplicating code.